stable/mlrun/templates/NOTES.txt

```diff
@@ -0,0 +1,21 @@
+1. Get the application URL by running these commands:
+{{- if .Values.ingress.enabled }}
+{{- range $host := .Values.ingress.hosts }}
+  {{- range .paths }}
+  http{{ if $.Values.ingress.tls }}s{{ end }}://{{ $host.host }}{{ . }}
+  {{- end }}
+{{- end }}
+{{- else if contains "NodePort" .Values.service.type }}
+  export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ include "mlrun.fullname" . }})
```

Comment:
```suggestion
  export NODE_PORT=$(kubectl --namespace {{ .Release.Namespace }} get service {{ include "mlrun.fullname" . }}) -o jsonpath="{.spec.ports[0].nodePort}"
```
- Prefer namespace arg first for consistency please (I also think it helps readability
- same for output format at the end

This everywhere

Reply:
I know this is not how we like to write commands, but since this was auto generated using `helm3 create mlrun` and it is like that in other official helm charts I prefer to leave it as is